customer-onboarding-showcase/readme.md

```diff
@@ -0,0 +1,31 @@
+# Customer Onboarding Showcase
+This module holds the code for the Customer Onboarding Showcase. It is a showcase of the capabilities of the Miranum Framework 
+(IDE as well as Connect) framework. In the figure below you can see the customer onboarding process as well as the integrations
+which are written using the Miranum Connect Framework. 
+
+![Customer-Onboarding-Process](./images/process.png)
+
+Currently, this showcase only supports Camunda Platform 7 as a process engine. This can be easily changed by migrating the process
+diagram for the customer-onboarding project to Camunda Platform 8. 
+Before startup make sure to start the Camunda Platform 7 from the [docker-compose file](./stack/c7/docker-compose.yml) 
+which can be found in the stack folder.
+To start this showcase build the project and run the following applications: 
+* customer-onboarding-mail-c7
+* customer-onboarding-starter-c7
+* risk-evaluation-c7
+* customer-onboarding-frontend
```

Comment:
I saw that we used to check in the ide config files. Wouldn't it be nice to create a compose run configuration and reference it here for the people who are using intellij which are likely to be almost all people. (Poor eclips user)

Reply:
You mean a docker-compose run configuration?